packages/x-components/src/x-modules/experience-controls/store/types.ts

```diff
@@ -0,0 +1,62 @@
+import { XEventsTypes } from '../../../wiring';
+import { XStoreModule } from '../../../store/store.types';
+import { XActionContext } from '../../../store';
+/**
+ * Experience Controls store state.
+ *
+ * @public
+ */
+export interface ExperienceControlsState {
+  /**
+   * Configuration for the `ExperienceControls` module.
+   */
+  controls: { [key: string]: unknown };
```

Comment:
Use `Dictionary<unknown>` instead (my bad)

```suggestion
  controls: Dictionary<unknown>;
```